src/components/Call/callState.js

```diff
@@ -60,7 +59,7 @@ function callReducer(callState, action) {
         clickAllowTimeoutFired: true,
       };
     case PARTICIPANTS_CHANGE:
-      const callItems = getCallItems(action.participants, callState.callItems);
```

Comment:
note that since callState is now longer dependent on the previous callState, i think we no longer need to keep this callState reducer around. instead, we can move all the state tracked in this file into the `Call` component as a few independent `useState` states, or perhaps factor them out into custom hooks. either way, i think the indirection of having to dispatch state changes to this reducer is no longer needed.

Reply:
🥳🥳🥳